### PR TITLE
added handout package

### PR DESCRIPTION
**Pull request by notestaff**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by notestaff** [comment]
@danijar I made a [conda-forge](https://conda-forge.org/) recipe for your Python package 'handout' .  Let me know if you want to be added as a co-maintainer of the recipe.

**Comment by danijar** [comment]
That looks great, thanks! I'm not very familiar with conda, so you don't need to add me.

File: recipes/handout/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py<35]
```
Comment (marcelotrevisani) [inline recipes/handout/meta.yaml]: I think this recipe can be noarch and you can select the python which you need on requirements
```suggestion
  noarch: python
```

File: recipes/handout/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (marcelotrevisani) [inline recipes/handout/meta.yaml]:
```suggestion
    - python >=3.5
```

File: recipes/handout/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/handout/meta.yaml]:
```suggestion
    - python >=3.5
```

File: recipes/handout/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/handout/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/handout/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - handout
+
+about:
+  home: "http://github.com/danijar/handout"
+  license: "Apache 2.0"
```
Comment (marcelotrevisani) [inline recipes/handout/meta.yaml]:
```suggestion
  license: "Apache-2.0"
```

**Pull request merged**